hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/HederaWorldState.java

```diff
@@ -309,6 +323,9 @@ public void commit() {
                     validateResourceLimit(creationCapacity, CONSENSUS_GAS_EXHAUSTED);
                 }
             }
+            final var consThrottleCapacityIsAvailable =
+                    Objects.requireNonNull(wrapped.recordsHistorian).throttleAllowsChildTransactions();
+            validateResourceLimit(consThrottleCapacityIsAvailable, CONSENSUS_GAS_EXHAUSTED);
```

Comment:
This might be misleading.
Is there a way to note which throttle has been hit? Or note a general non gas throttle?

Reply:
We would need to add a new protobuf response code, but it might be worth it?

Maybe, `CHILD_TRANSACTIONS_WERE_THROTTLED`?

Reply:
What do you think @Neeharika-Sompalli? Worth having a new response code?

Reply:
@tinker-michaelj I was wondering the same , but saw description for `CONSENSUS_GAS_EXHAUSTED` and that seemed reasonable
``
  /**
   * Consensus throttle did not allow execution of this transaction. System is throttled at
   * consensus level.
   */
```

Reply:
Reasonable for now but we fix in 0.40?

Reply:
It says throttled in handle. Why would we need a new response code? Precompiles have only one handle throttle right ?

Reply:
Sounds good. 👍

Reply:
Since, it says throttles in handle, the response code seems correct to me( by its description). If we still need a new response code, we could do in 0.40.